python/paddle/fluid/layers/nn.py

```diff
@@ -7455,6 +7456,58 @@ def maxout(x, groups, name=None):
     return out
 
 
+def reorg(x, stride, name=None):
+    """
+    Gives a stride to reorg the input tensor
+
+    Here are some example:
+
+    input is 4D LoDtensor with shape [batch, channel, height, width] and has an attrs stride = 2
+
+    reorg will do some math work to reorder the elements of input according to stride to construt
```

Comment:
what's the "math"?

Reply:
the formula is too complex to show and I add the more specific process of how this op work in latest commit